platform/engineering/official_github_teams_process

```diff
@@ -0,0 +1,38 @@
+# GitHub Teams on for VA.gov
+
+## What is GitHub Teams, and why are we talking about it?
+
+GitHub Teams give organizations the ability to create groups of members and control access to repositories. Team members can be granted read, write, or admin permissions to specific repositories.
+For more information on GitHub teams, please see [GitHub docs](https://help.github.com/en/enterprise/2.19/admin/user-management/creating-teams).
+
+At the moment, some teams have their own GitHub Teams, while others do not. GitHub Teams’ benefits and functions are not fully understood across VA.gov. Our goal is to provide a basic understanding of GitHub teams, establishing a logical structure of teams first. Then we can improve the process of granting access and developers’ experience on VA.gov. 
+
+In the future, we will be using *Official GitHub Teams* as a source of truth for team rosters, access, and auditing purposes for VSP. The process of creating and maintaining Official GitHub Teams is below. 
+
+## GitHub Team Process
+
+### Creating a GitHub Team 
+
+* Work with your DEPO, Tech, and Team Lead to change an existing team to meet the requirements below or [create](https://help.github.com/en/enterprise/2.19/admin/user-management/creating-teams)  an *Official GitHub Team* on VA.gov. 
+    * All *Official GitHub Team* names should start with `vagov-` followed by the team name, which indicates the type of team this is. Examples of this are `vfs-*` (Veteran-facing Services) for teams working on an application or building tools and services on VA.gov.  Another is `vsp-*` (Veteran-facing Services Platform) for teams working on the platform supporting VFS teams. Examples would be `vagov-vsp-operations or `vagov-vfs-public-website`. If those two team types do not fit your team, work with the Platform team to create a logical team type prefix. 
+    * Select the team name after `vfs-*` or `vsp-*`.  An example would be `vagov-vfs-[team name here]`. Choose a name that will not change and is approved by DEPO. Team names should be connected to the team’s work and not the contractor working on the contract. 
+    * Each GitHub team needs to have DEPO one Maintainer; these are Admins on a team who can add and remove users. The Maintainer should be the team’s DEPO lead. 
+    * Maintainers should add all members to the team including both engineers and nonengineers. 
```

Comment:
If we are going to use this for permissions, is there specific guidance we might have for a team who's wanting to grant access to specific types of tools to only certain members? i.e. only developers should have approval power on the codebase

Reply:
Hmm, maybe, I would reply on your advice here. Maybe there can be a nested Dev Team?